_policies/01_policies.md

```diff
@@ -58,32 +60,32 @@ Additionally, articles should include clear links to, and mention of, the releva
 
 For an example of an article hosted on GitHub in this style (though not a LiveCoMS article), see [the following perpetual review](https://github.com/MobleyLab/benchmarksets). 
 
-
-One written, the authors post the resulting PDF document to a preprint server of their choice, which can be any one of:
+Ultimately, assuming your article passes peer review, LiveCoMS will be hosting a published version of your article, and you will have what essentially constitutes a preprint available on GitHub as well.
+However, you are also free to post your article to standard preprint servers, such as:
 * [ArXiv](https://arxiv.org)
 * [BioRxiv](http://www.bioarxiv.org)
 * [ChemRxiv](https://chemrxiv.org/)
 * [Faculty of 1000 Research](https://f1000research.com)
 * Any of the [Open Science Framework preprint servers](https://osf.io/preprints/) (engrXiv, etc.)
 
-Posting to the preprint server can be done at any time prior to submission. 
-
 # Submission and the Review Process
 
 
 ## Article submission 
 
-After posting to the preprint server, the author uploads the final
+When ready for submission, the author uploads the final
 article PDF (created using the [LiveCoMS
 templates](http://www.github.com/livecoms/author_templates), discussed above) and a link to the GitHub site for the article.
 
 The authors must also submit a cover letter with suggestions for 4--5
 reviewers, and note any deviations from the presubmission letter.
 
+Article submission also involves paying a nominal charge of of $100 per submission, which covers our peer review management system as well as ongoing operation costs (web hosting, etc.).
```

Comment:
I thought we would do $50?

Reply:
Google Doc says $100. https://docs.google.com/document/d/1E2BhDiwRO9bCNnDWHGSJ-skVyTVYjNKwNHID0B4dR5Y/edit

Reply:
We could try to have a quick(!) discussion on the amount at next editorial meeting.  I forgot if we ever figured out a way to bank the excess over the Scholastica charge.

Reply:
Right now, you have to literally 'bank' it.  It goes back into your bank.  you can only credit people by reimbursing them.  I can ask a bit more about if they see a possibility of changing it so that you can have a "reserve" account.